src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -418,6 +581,73 @@ export function DistributionForm({
     return `Avail. ${commify(round(humanNumber, 2))}`;
   };
 
+  const getVaultOptions = (options?: {
```

Comment:
@bark4mark why do we need two toggles here? under what conditions should `includeHedgey` and `includeConnectWallet` have different values?

Reply:
never mind, i understood it after digging into the code more. please check out my changes to the method & let me know if i'm missing somethign